promql/functions.go

```diff
@@ -478,6 +478,34 @@ func funcSumOverTime(ev *evaluator, args Expressions) model.Value {
 	})
 }
 
+// === stddev_over_time(matrix model.ValMatrix) Vector ===
+func funcStddevOverTime(ev *evaluator, args Expressions) model.Value {
+	return aggrOverTime(ev, args, func(values []model.SamplePair) model.SampleValue {
```

Comment:
Any reason for not having the inner function as a separate helper function so that it can be reused in funcStdvarOverTime? It would also read more easily, I guess.